docs/guide/learning/books.md

```diff
@@ -175,3 +175,19 @@ Readers will then use Laravel to set up a web service and integrate the front en
 </useful-links-section>
 </useful-links>
 
+### Pro Vue.js 2
+**Author:** Adam Freeman
+**Publisher:** Apress
+
+> From beginner to pro guide to Vue.js written in simple language by best-selling author Adam Freeman.
+
+Explore Vue.js to take advantage of the capabilities of modern browsers and devices using the fastest-growing framework for building dynamic JavaScript applications. You will work with the power of the Model-View-Controller (MVC) pattern on the client, creating a strong foundation for complex and rich web apps.
```

Comment:
Will you approve the following?

_Book begins with the crash course during you'll build eshop app using vue-cli, Vuex, router, devtools etc.. Even if you stop reading after it, you will've strong overview how powerful Vue.js is. The rest of the book explains all these parts of Vue.js in deep detail but with easy to understand and always realistic examples. Unlike other books, Pro Vue.js 2 use from the first moment real-word development based on tools and libs like vue-cli, eslint, Axios etc._

Reply:
How about this? 

_The book begins with a crash course, during which you'll build an e-shop app using Vue CLI, Vuex, Vue Router and more. This alone gives a good base knowledge about how powerful Vue.js is. 

The rest of the book covers Vue.js in deep detail, but with easy to understand and always realistic examples. One thing to note, Pro Vue.js 2 uses real-word development tools and libs, like Vue CLI, ESLint, Axios from the beginning, so you can get to know them from the start._